docs/protocol/network/data-structures/transaction.md

```diff
@@ -0,0 +1,136 @@
+
+| Field   |  Type  | Description                         |
+| ------- | :----: | ----------------------------------- |
+| `pkh`   | `[u8]` | Slice of public key hash (20 bytes) |
+| `value` | `u64`  | Transaction value                   |
+
+### Data Request outputs
+
+Data request outputs are used to define how to request the data from Witnet.
+
+| Field              |  Type  | Description                                                                                                                     |
+| ------------------ | :----: | ------------------------------------------------------------------------------------------------------------------------------- |
+| `data_request`     | `[u8]` | Data request script as a byte array                                                                                             |
+| `value`            | `u64`  | Transaction value that will be used as reward to be distributed after consensus has been reached and fees have been substracted |
+| `witnesses`        |  `u8`  | Minimum amount of data request witnesses                                                                                        |
+| `backup_witnesses` |  `u8`  | Number of backup data request witnesses                                                                                         |
+| `commit_fee`       | `u64`  | Miner fee for each valid commit output included in the block during the **commit phase**                                        |
+| `reveal_fee`       | `u64`  | Miner fee for each valid reveal output included in the block during the **reveal phase**                                        |
+| `tally_fee`        | `u64`  | Miner fee for each valid P2PKH output included in the block during the  **tally phase**                                         |
+| `time_lock`        | `u64`  | The UTC Unix timestamp after which data request may be executed                                                                 |
+
+The minimum data request reward to be distributed among nodes that agreeed with the consensus is defined as follows:
+
+```math
+dr_reward_min = value - (witnesses * commit_fee) - (witnesses * reveal_fee) - (witnesses * tally_fee)
+```
+
+During the reveal and tally phase, some committers may not participate in the data request.
+However, the block miner is incentivized to include as many outputs as committers as possible in order to increase the block fees.
+
+### Commit outputs
+
+| Field        |  Type  | Description                                                                                                |
+| ------------ | :----: | ---------------------------------------------------------------------------------------------------------- |
+| `commitment` | `[u8]` | Signed hash of the data request result by an individual node                                               |
```

Comment:
Shall we specify that the commitment is nonced to prevent lazy commitment / replaying?

Reply:
Makes sense